intervention_compassion/models/compassion_intervention.py

```diff
@@ -592,7 +597,7 @@ def create_commitment(self):
             'res_model': 'compassion.intervention.commitment.wizard',
             'context': self.with_context({
                 'default_intervention_id': self.id,
-                'default_commitment_amount': self.hold_amount,
+                'default_commitment_amount': self.commitment_amount,
```

Comment:
Here, the fix is this one: we want to make sure we don't commit more money than what the intervention costs that day in USD dollars. So we want to refresh the intervention first and commit maximum of intervention amount.
```suggestion
                'default_commitment_amount': max(self.hold_amount,
                                                                          self.estimated_costs),
```